akka-http-core/src/main/scala/akka/http/impl/engine/HttpConnectionTimeoutException.scala

```diff
@@ -0,0 +1,5 @@
+package akka.http.impl.engine
+
+import scala.util.control.NoStackTrace
+
+class HttpConnectionTimeoutException(msg: String) extends RuntimeException(msg) with NoStackTrace
```

Comment:
Technically it should be "2009-2015", and we'll update the headers to be "2009-2016" once new years hits the calendar.